src/components/forms/buffered-input-hoc.jsx

```diff
@@ -39,6 +39,11 @@ export default function (Input) {
             this.setState({value: null});
         }
         handleChange (e) {
+            const isNumeric = typeof this.props.value === 'number';
```

Comment:
So... the top of this file says `DO NOT EDIT`...

Maybe add a comment about why it is being edited in the face of such clear instructions ;)

Reply:
lol my b thanks!